src/libraries/System.Net.Quic/src/System/Net/Quic/Implementations/MsQuic/Interop/MsQuicTlsSupportHelper.Unix.cs

```diff
@@ -0,0 +1,10 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+namespace System.Net.Quic.Implementations.MsQuic.Internal
+{
+    internal static partial class MsQuicTlsSupportHelper
+    {
+        public static bool IsTls1_3Disabled() => false;
```

Comment:
Could we just `#if ... ` instead of putting it into extra helper class and files?

Reply:
Good point, I went a little OTT there.